index.js

```diff
@@ -1,68 +1,31 @@
 'use strict';
+
 const writeFile = require('broccoli-file-creator');
 const mergeTrees = require('broccoli-merge-trees');
 
-const dummyComponent = `
-  import Component from '@ember/component';
-
-  export default Component.extend({
-
-  });
-`;
-
-const dummyService = `
-  import Service from '@ember/service';
-  export default Service.extend({
-  });
-`;
-
-
-const dummyInitializers = `
-  export default {
-    name: 'ember-accessibility',
-    initialize() {}
-  };
-`
-
-const files = {
-  'initializers/ember-accessibility.js': dummyInitializers,
-  'components/accessibility-result.js': dummyComponent,
-  'components/accessibility-tester.js': dummyComponent,
-  'components/critical-icon.js': dummyComponent,
-  'components/minor-icon.js': dummyComponent,
-  'components/moderate-icon.js': dummyComponent,
-  'components/serious-icon.js': dummyComponent,
-  'components/toggle-result.js': dummyComponent,
-  'services/accessibility-test.js': dummyService
-};
+const dummyFiles = require('./utils/dummy-files');
 
+let testAccessibility = process.env.TEST_ACCESSIBILITY;
 let filesTree = [];
 
 module.exports = {
   name: require('./package').name,
 
   treeForApp(tree) {
-    let environment = this.app.env;
-
-    let shouldExcludeFiles = environment === 'production';
-
-    if (shouldExcludeFiles) {
-      for (let file in files) {
-        filesTree.push(writeFile(file, files[file]));
+    if (!testAccessibility) {
```

Comment:
Shall we exclude in production build by default?

Reply:
If they want to bundle in production, then they pass `testAccessibility = true` while building

Reply:
I am not sure I understand. This does exclude our code from production builds(or for that matter, any environment) by default. It will get bundled only when the env is set to true.

Reply:
Sorry, I didn't check !testaccessibility, Its fine 👍

Reply:
Shall we merge?